web/src/app/models/userSocial.model.ts

```diff
@@ -6,4 +6,6 @@ export interface UserSocial {
   youtube?: Social;
   instagram?: Social;
   uid: string;
+  id?: string;
```

Comment:
what is the difference between `uid` and `id`

Reply:
id is the userSocialDocumentID and uid is the userID to which the userSocialDocument is linked

Reply:
Ah I see, can the reference ID be `userId`, then it is obvious it is a reference

Reply:
Okay @eddiejaoude